autotest/ogr/ogr_hana.py

```diff
@@ -1170,6 +1170,94 @@ def create_feature(fid, geom_wkt=None):
     layer.CommitTransaction()
 
 
+###############################################################################
```

Comment:
I think, we should also adjust
def test_ogr_hana_1():
    def test_capabilities(update, capabilities):

Reply:
It seems like that test_ogr_hana_1() tests the advertised capabilities of the data source, not layer capabilities like OLCFastGetExtent.
Should we add another test for this?

Reply:
Ah, this is the layer capabilities. I forgot that we test it in create_tpoly_table. So, you don't need to do anything.